src/azure-cli-core/azure/cli/core/tests/test_adal_authentication.py

```diff
@@ -4,8 +4,12 @@
 # --------------------------------------------------------------------------------------------
 
 # pylint: disable=line-too-long
+import datetime
 import unittest
-from azure.cli.core.adal_authentication import _try_scopes_to_resource
+import unittest.mock as mock
+from unittest.mock import MagicMock
```

Comment:
Can be improved.

Reply:
How?

Reply:
Delete 2nd line or delete 1st line and import all used class in mock